src/App.js

```diff
@@ -1,22 +1,39 @@
 import "./App.css";
 import React, { useRef, useEffect, useState } from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import Dashboard from "./components/Dashboard.js";
+import LoginForm from "./components/LoginForm";
 // eslint-disable-next-line import/no-webpack-loader-syntax
-import Map from "./components/Map.js";
-import SurfSpotCard from "./components/SurfSpotCard";
 
 <link
   rel="stylesheet"
   href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
 />;
 
+const testUser = {
+  email: "carlo@dumb.com",
+  password: "imstupid",
+};
 
 function App() {
-  return (
-    <div>
-      <Map />
-      <SurfSpotCard />
-    </div>
-  );
+  const [user, setUser] = useState({ email: "" });
+  const [error, setError] = useState("");
+  const Login = (details) => {
+    console.log(details);
+    if (details.email === testUser.email && details.password === testUser.password) {
```

Comment:
Might be better to generalize testUser and then hardcode the test user in elsewhere?